python/whylogs/api/store/sqlite_store.py

```diff
@@ -0,0 +1,126 @@
+import logging
+import os
+import sqlite3
+from datetime import timedelta, timezone
+
+from whylogs.api.store import ProfileStore
+from whylogs.api.store.query import BaseQuery, DatasetIdQuery, DateQuery
+from whylogs.core import DatasetProfile, DatasetProfileView
+
+logger = logging.getLogger(__name__)
+
+
+class SQLiteStore(ProfileStore):
+    def __init__(self):
+        self._db_location = os.getenv("SQLITE_STORE_LOCATION")
+        if self._db_location is None:
+            raise ValueError("You must define a store with the SQLITE_STORE_LOCATION env var")
+        self.conn = sqlite3.connect(database=self._db_location, check_same_thread=False)
+        self.cur = self.conn.cursor()
+
+    def __enter__(self) -> "SQLiteStore":
+        return self
+
+    def __exit__(self, exception_type, exception_value, traceback) -> None:
+        try:
+            self.cur.close()
+            self.conn.close()
+        except exception_type:
+            logger.error(traceback)
+            raise exception_value
+
+    def __del__(self) -> None:
+        try:
+            self.cur.close()
+            self.conn.close()
+        except Exception as e:
+            logger.debug(f"Connection not established. Error: {e}")
+
+    def _init_db(self):
+        init_db_query = "CREATE TABLE IF NOT EXISTS profile_store(id string, date datetime, profile blob);"
+        self.cur.execute(init_db_query)
+
```

Comment:
We should be able to just init this once right? No need to execute an additional query before all of the real work, even if it would be a no-op

Reply:
right! will move it onto the init method